README.md

```diff
@@ -23,3 +23,11 @@ By removing errors and inconsistencies, the goal is to improve performance of th
 We would like to thank the authors of the Cleaned Alpaca dataset for their effort.
 
 We would like to thank the original creators of the Alpaca datasets for making their data available to the public.
+
+# Licensing
+
+The Cleaned German Alpaca Dataset is licensed under [CC BY NC 4.0](DATA_LICENSE).
```

Comment:
Should we use an absolute link here? Like below?

Reply:
Definitely, I changed it now!